docs/_docs/Instructors/01-Teaching-process.md

```diff
@@ -0,0 +1,173 @@
+
+* Jupyter notebooks: almost all content is delivered through jupyter notebooks. 
+This provides an interactive way of learning, as students can run code while reading
+through the explanations. 
+* Nbgrader: In the context of jupyter notebooks, we also perform evaluation through
+graded notebooks. For this purpose we use `nbgrader` (an overview of how to use it
+can be found in [Using nbgrader](./03-Using-nbgrader-for-Exercise-Notebooks.md).)
+
+All materials are planned, built and reviewed through github. In particular all repositories
+are found under the [Academy GitHub](https://github.com/LDSSA/). These are the main
+repositories to check and use:
+* [curriculum-development](https://github.com/LDSSA/curriculum-development) (public): Our curriculum is reviewed and updated in this repository 
+* [batchX-instructors](https://github.com/LDSSA/batch6-instructors) (private): To prepare the materials beforehand, instructors work in this
+private repository. It's in this repo that all review and discussion around specific materials 
+happens - for this purpose all feedback and suggestions should be posted as issues 
+* [batchX-students](https://github.com/LDSSA/batch5-students) (public):  Close to the deadline of the unit, we trigger a process that releases the content of 
+ the instructors repository into this public repository; this is the repository students will access
+
+Following the patterns above you can check any previous edition's or the current batch materials.
+
+# Areas of responsibility
+ 
+You can help improve the units by joining one of several areas of responsibility. Below 
+there is an overview of how they interact and what are the key responsibilities you
+can have within each
+
+<img src="../../images/aors-teaching-process.png"/>
+
+### Curriculum
+
+Overall responsibilities:
+* Define which contents should be offered
+
+
+### Teaching
+
+Overall responsibilities:
+
+* Produce and improve materials
+* Release content and provide extra explanations and async support during the specialization
+* Support teams during hackathons (if the unit is in the context of a specialization)
+
+There are a few different roles worth mentioning:
+
+| Role | Description |
+|------|-------------|
+| Admission/Bootcamp instructor | Build/improves units and delivers presentation on bootcamp weekend (admissions units are excluded from this)|
+| Specialization instructor | Build/improves specialization units/hackathon and provides support during the hackathon day |
+| Specialization lead | Coordinates specialization and potentially participates in minor tasks, if necessary |
+| Specialization mentor | Typically a senior, provides support to other instructors developping the specialization |
+
+
+### QA
+
+Overall responsibilities:
+* Review and test materials
+* Deliver early feedback for instructors
+
+
+### Student Success
+
+Overall responsibilities:
+* Keep track of students progress 
+* Collect and deliver student feedback
+
```

Comment:
I would add operations to coordinate office hours

Reply:
I think that's out of scope for the purpose of  impacting the materials, let me put a pin on that and review when all of the section is revamped maybe

Reply:
sounds good to me!